ios/PurchasesHybridCommon/PurchasesHybridCommon/StoreProduct+HybridAdditions.swift

```diff
@@ -21,8 +21,8 @@ import StoreKit
             "introPrice": NSNull(),
             "price": self.price,
             "priceString": self.localizedPriceString,
-            "productCategory": self.productCategoryString,
-            "productType": self.productTypeString,
```

Comment:
I guess this is a breaking change for javascript hybrids? We should make sure to document it.

Reply:
+1, this is a breaking change, right? 
Like, you could reasonably have some code that used this value to determine whether the product was a consumable and treat it differently, and it'd break if you upgrade, right?